Cargo.toml

```diff
@@ -25,6 +25,7 @@ signal-hook = "0.3.14"
 mlua =  { version = "0.8.7", features = ["lua54", "send", "vendored"] }
 regex = "1.7.1"
 strip-ansi-escapes = "0.1.1"
+vte = "0.11.0"
```

Comment:
quick note: I added this `vte` dependency to be able to parse out terminal control codes in `printable_chars.rs`. However, it is [already a transitive dependency via the `strip-ansi-escapes` crate](https://github.com/luser/strip-ansi-escapes/blob/a0a306ab6c5a76f269fea8659fdd736f1d2d56b9/Cargo.toml#L13-L14) so there's no increase in the net size of deps.

Reply:
No problem